### PR TITLE
fix for raw query

### PR DESCRIPTION
**Pull request by egorpro123**

### CODE REVIEW

File: lib/adapter.js [hunk 1]
```diff
@@ -113,21 +111,25 @@ module.exports = (function main() {
...
-
-    query(collectionName, statement, data, callback) {
-      if (_.isFunction(data)) {
-        callback = data;
-        data = null;
+    teardownSync() {
+      if (dbConnected) {
+        try {
+          db.closeSync();
+// eslint-disable-next-line no-empty
```
Comment (vedi) [inline lib/adapter.js]: it's bad idea to mute it (http://stackoverflow.com/questions/1234343/why-are-empty-catch-blocks-a-bad-idea)

**Review by vedi — CHANGES_REQUESTED** [review]
you mixed up refactoring and meaningful changes in one PR, I cannot get what's related to "raw query". Could you help?

File: lib/adapter.js (hunk 1 above)
Comment (egorpro123) [reply lib/adapter.js]: I do not know how else to suppress all errors

**Comment by egorpro123** [comment]
It was not refactoring, it was necessary to transfer because when working raw query the function _query was wrapped and the wrong parameters were passed to it.

File: lib/adapter.js (hunk 1 above)
Comment (vedi) [reply lib/adapter.js]: Did not a link help?

**Comment by vedi** [comment]
got it
fix other comment and we're ready to go ahead

File: lib/adapter.js (hunk 1 above)
Comment (egorpro123) [reply lib/adapter.js]: What kind of help? I found this:
`} catch (e) {
        return null;
    }`

Comment (vedi) [reply lib/adapter.js]: http://stackoverflow.com/a/28941060/1204329

> 1. An empty catch block defeats the purpose of exceptions
> 2. At the very least, the catch block should contain a comment explaining why it is appropriate to ignore the exception.

Comment (egorpro123) [reply lib/adapter.js]: I'll add a comment

**Pull request merged**